assets/app/scripts/controllers/overview.js

```diff
@@ -241,20 +241,37 @@ angular.module('openshiftConsole')
 
 
     var associateDeploymentConfigTriggersToBuild = function(deploymentConfig, build) {
+      // Make sure we have both a deploymentConfig and a build
       if (!deploymentConfig || !build) {
         return;
       }
+      // Make sure we have a build output
+      if (!build.parameters.output.to) {
```

Comment:
build.parameters.output.dockerImageReference is also a possibility

@ironcladlou do deployment config image triggers only trigger off of image repositories?  or can they trigger off of a docker registry directly
@bparees if a build's output refers to dockerImageReference then that is not an image repository correct, it pushes directly to a docker registry?